### ISSUE TITLE
Make sure Basis looks and works decently when used as an admin theme.

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#2653]
This came up in https://github.com/backdrop/backdrop-issues/issues/1725#issuecomment-296590390

>>Minor styling issue (tested in Chrome on Mac): The position of the X-Button doesn't align well, see screenshot:
>The CSS thing is a Basis theme problem. Recommend open a new issue please.

I'm opening this issue here because I could not reproduce the styling problem on a vanilla 1.x installation. At the time of writing this, we do not have a way to invoke a pop-up dialog in the front-end, so what I've done is switch the admin theme to Basis, and navigated to the manage blocks UI in order to see what a dialog looks like. Here's how that page is being rendered:

![image](https://cloud.githubusercontent.com/assets/2423362/25365117/1250da7e-29aa-11e7-8576-4fff1db2a098.png)

...ignoring this mess for a moment in order to verify the issue with the dialog styling, I just hit one of the "Add block" buttons and here's what the dialog looks like:

![image](https://cloud.githubusercontent.com/assets/2423362/25365177/60d2033a-29aa-11e7-9005-712d73c6b92e.png)

...same with the "Configure block" dialogs; no styling issues:

![image](https://cloud.githubusercontent.com/assets/2423362/25365222/a778a168-29aa-11e7-95d5-b23801887bf5.png)

With #1725 we'll be introducing something new: dialogs, which are a component usually (up until now) used in the backend *only*, are now to be used in the front-end too. So, the point is that we should make sure that Basis (and any theme that ships with core really) looks and works decently when used as an admin theme.

**Comment by olafgrabienski** [comment]
Basis is meant as a frontend theme, and I guess improving it to be used as an admin theme would be a huge task. Shouldn't we limit the task to former 'backend only' components like the modal dialog?

**Comment by klonos** [comment]
Sure, but it doesn't have to be perfect; just decent and usable. As you can see from the screenshot above, the manage blocks UI is terribly broken.

**Comment by olafgrabienski** [comment]
> it doesn't have to be perfect; just decent and usable

Yes, that would be nice, absolutely. Let's however avoid the impression that Basis is an admin theme alternative.

**Comment by klonos** [comment]
> Let's however avoid the impression that Basis is an admin theme alternative.

@olafgrabienski there's quite a lot of people that switch their sites to using the same theme as front/back-end. The reason is that when authors are editing nodes, they can be confused to see another theme. Some even get the impression that the site is broken. I've had some use cases with small sites (where the owner is the sole author of content and they happen to not be technically-savvy) where they kept asking "Why aren't the changes we've discussed applied everywhere? You only seem to have fixed the front page."

Some feel the opposite way and say that it is a good thing to be switching to another theme when in edit mode. So the fact is that so long as there is an option that allows people to switch to Basis as an admin theme, some people will do just that. Telling them "Yeah, but it is not *intended* to be used as an admin theme" would make them say "Then please remove it from being an option in the admin theme drop-down". That's why in the issue summary I said:

> ...the point is that we should make sure that Basis (**and any theme that ships with core really**) looks and works decently when used as an admin theme.

And I'm sure the other way around applies where some might be using backend-intended themes like Seven as front-end themes.

Besides, I strongly remember that when @wesruv was developing Basis kept asking opinions on things that dealt with the backend, like the drag handler on draggable table rows (he wanted to replace the cross with something that resembled the 3 horizontal lines we use in the hamburger mobile menu). He in fact seems to have actually themed tables in Basis 👍 :

![image](https://cloud.githubusercontent.com/assets/2423362/25457094/68b77156-2b18-11e7-8f14-bbc1de978a42.png)

**Comment by docwilmot** [comment]
Absolutely agree with @klonos here. If Basis is that much broken as an admin theme it definitely has to be fixed. Marking as bug.

**Comment by olafgrabienski** [comment]
I agree too, @klonos . Changed my mind, even if I wouldn't recommend to use Basis as an admin theme. (Guess I'm one of the "it is a good thing to be switching to another theme when in edit mode" guys, but I see there other approaches.)

**Comment by stpaultim** [comment]
If this one is addressed, let's be sure to look at the specific issue raised here:
https://github.com/backdrop/backdrop-issues/issues/2259

Problems with Manage Blocks page.
